features/bevragen/persoon/verblijfstitel/dev/verblijfstitel-gba.feature

```diff
@@ -34,17 +34,44 @@ Achtergrond:
     | aanduiding.code         | 37                                                                             |
     | aanduiding.omschrijving | Vw 2000 art. 8, onder e, gemeenschapsonderdaan econ. niet-actief, arbeid spec. |
 
-  Scenario: verblijfstitel heeft een datum einde
+  Scenario: verblijfstitel heeft een datum einde in het verleden
     Gegeven de persoon met burgerservicenummer '000000140' heeft de volgende 'verblijfstitel' gegevens
-    | aanduiding verblijfstitel (39.10) | datum ingang verblijfstitel (39.30) | datum einde verblijfstitel (39.20) |
-    | 37                                | 19980201                            | 20020701                           |
+    | aanduiding verblijfstitel (39.10) | datum ingang verblijfstitel (39.30) | datum einde verblijfstitel (39.20) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) |
+    | 37                                | 19980201                            | 20020701                           | 100000                          | 20230127                       |
     Als gba personen wordt gezocht met de volgende parameters
     | naam                | waarde                          |
     | type                | RaadpleegMetBurgerservicenummer |
     | burgerservicenummer | 000000140                       |
-    | fields              | verblijfstitel.aanduiding       |
+    | fields              | verblijfstitel                  |
     Dan heeft de response een persoon zonder gegevens
 
+  Scenario: verblijfstitel heeft een datum einde vandaag
+    Gegeven de persoon met burgerservicenummer '000000140' heeft de volgende 'verblijfstitel' gegevens
+    | aanduiding verblijfstitel (39.10) | datum ingang verblijfstitel (39.30) | datum einde verblijfstitel (39.20) | aanduiding in onderzoek (83.10) | datum ingang onderzoek (83.20) |
+    | 37                                | 19980201                            | vandaag                            | 100000                          | 20230127                       |
```

Comment:
Is er een reden / is het dan handig om ook in onderzoek onderdeel te maken van dit scenario?

Reply:
ik wil checken dat geen enkel gegeven van verblijfstitel geleverd wordt wanneer datum einde vandaag. Dus niet alleen de aanduiding, of alleen de datumEinde, maar alles van de verblijfstitel